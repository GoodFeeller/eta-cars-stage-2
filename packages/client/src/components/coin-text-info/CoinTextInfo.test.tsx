import { render, screen } from '@testing-library/react'
import { ICoin } from '../../types/ICoin'
import CoinTextInfo from './CoinTextInfo'

const coin: ICoin = {
  rank: '1',
  symbol: 'BTC',
  name: 'Bitcoin',
  id: 'bitcoin',
  priceUsd: '21000.2313125256326',
  maxSupply: '1000000.223',
  supply: '141312',
  marketCapUsd: '32131412123.231242412321',
  changePercent24Hr: '0.24',
}
test('CoinTextInfo snapshot test', () => {
  render(<CoinTextInfo coin={coin} />)
  const info = screen.queryByTestId('info')
  expect(info).toMatchSnapshot()
})
