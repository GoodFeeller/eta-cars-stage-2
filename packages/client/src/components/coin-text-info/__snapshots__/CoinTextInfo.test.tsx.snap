// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoinTextInfo snapshot test 1`] = `null`;
